pages/am/3.x/user-guide/user-management/user/user-guide-users.adoc

```diff
@@ -8,11 +8,117 @@
 
 === List users
 
-To search users, navigate to `Users` section of your domain's settings. The input search box will let you find user by its username.
+To search users, navigate to `Users` section of your domain's settings and set a search query expression.
+You can also list users via the `/users` endpoint of the AM management API.
```

Comment:
To search for users, navigate to the *Users* section of your domain's settings and enter the username in the search field.